services/QuillLMS/client/app/bundles/Teacher/components/unitTemplate/unitTemplateSelectedActivitiesTable.tsx

```diff
@@ -26,10 +26,10 @@ export const UnitTemplateSelectedActivitiesTable = ({ activities, selectedActivi
   function renderHeaders() {
     return(
       <tr className="data-table-headers">
-        {unitTemplateDataTableFields.map(header => {
+        {unitTemplateDataTableFields.map((header, i) => {
           const { width, name } = header;
           const style = { width: `${width}`, minWidth: `${width}` }
-          return <th className="data-table-header" scope="col" style={style}>{name}</th>;
+          return <th className="data-table-header" key={i} scope="col" style={style}>{name}</th>;
```

Comment:
I don't think it's likely to matter in this use-case, but what I've read suggests that using index values for `key` can lead to unexpected behavior (since the order of an array can change, but the keys will remain the same, which will signal to React that it does not need to re-render).